### PR TITLE
FIX wrong shortcut key for macintosh

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

File: htdocs/main.inc.php [hunk 1]
```diff
@@ -2575,17 +2575,24 @@ function top_menu_quickadd()
 {
 	global $conf, $langs;
 
+	$tmp = getBrowserInfo($_SERVER["HTTP_USER_AGENT"]);
+	$ostype = $tmp['browseros'];
```
Comment (fappels) [inline htdocs/main.inc.php:2579]: ostype is already in $conf->browser->os

Comment (hregis) [reply htdocs/main.inc.php]: @fappels good news, great, it's fixed, thank you

File: htdocs/main.inc.php [hunk 2]
```diff
@@ -2825,12 +2837,16 @@ function top_menu_bookmark()
 	// accesskey is for Windows or Linux:  ALT + key for chrome, ALT + SHIFT + KEY for firefox
 	// accesskey is for Mac:               CTRL + key for all browsers
 	$stringforfirstkey = $langs->trans("KeyboardShortcut");
-	if ($conf->browser->name == 'chrome') {
-		$stringforfirstkey .= ' ALT +';
-	} elseif ($conf->browser->name == 'firefox') {
-		$stringforfirstkey .= ' ALT + SHIFT +';
-	} else {
+	if ($conf->browser->os === 'macintosh') {
```
Comment (eldy) [inline htdocs/main.inc.php:2840]: Do you mean shortcut is CTL on mac, even on firefox ?

Comment (hregis) [reply htdocs/main.inc.php:2840]: Hello @eldy 
yes there is no ALT key on the mac.
CTL + key is possible (shorter)
CTL + SHIFT + key is possible too

**Pull request merged**